builtin/game/chatcommands.lua

```diff
@@ -1056,3 +1056,23 @@ core.register_chatcommand("kill", {
 		return handle_kill_command(name, param == "" and name or param)
 	end,
 })
+
+core.register_chatcommand("whereis", {
+	params = "[<name>]",
+	description = "Show location of yourself or another player",
+	privs = {kick = true, ban = true},
```

Comment:
Why these privs? Moderators on my survival server often tell me they don't want privs that allow unfair gameplay advantages. Being able to locate a player would give them this.

Reply:
Good point. But it's still provided by the IRC mod without any restrictions, and I feel that it should definitely have some sort of restriction. Should I make it require `server` like `/kill`?